### PR TITLE
remove deprecated annotations

### PR DESCRIPTION
**Pull request by garak**

### CODE REVIEW

**Comment by stof** [comment]
Would be good to take the other part of my comment (https://github.com/KnpLabs/KnpMenuBundle/pull/412#discussion_r321779758) into account too.

File: tests/DependencyInjection/Compiler/AddExtensionsPassTest.php [hunk 1]
```diff
@@ -102,6 +100,8 @@ public function testMissingAddExtension()
             ->method('getParameterBag')
             ->willReturn($parameterBagMock);
 
+        $this->expectException(\Symfony\Component\Config\Definition\Exception\InvalidConfigurationException::class);
```
Comment (derrabus) [inline tests/DependencyInjection/Compiler/AddExtensionsPassTest.php]: That's probably a matter of taste, but we could now import the class as well.

Comment (stof) [reply tests/DependencyInjection/Compiler/AddExtensionsPassTest.php]: indeed, please do it.

Comment (garak) [reply tests/DependencyInjection/Compiler/AddExtensionsPassTest.php]: Done!

**Pull request merged**